optimum/onnxruntime/modeling_ort.py

```diff
@@ -1545,6 +1559,341 @@ def _prepare_onnx_outputs(self, onnx_outputs):
         return outputs
 
 
+AUDIO_CLASSIFICATION_EXAMPLE = r"""
```

Comment:
Probably you already have - but can you make sure the code snippet works?

Reply:
Done